.travis.yml

```diff
@@ -28,6 +28,8 @@ install:
    - cd ~/catkin_ws
 
 before_script: # Use this to prepare your build for testing e.g. copy database configurations, environment variables, etc.
-  - source /opt/ros/$ROS_DISTRO/setup.bash  
+  - source /opt/ros/$ROS_DISTRO/setup.bash
 script: # All commands must exit with code 0 on success. Anything else is considered failure.
   - catkin_make -j2
+  - source ~/catkin_ws/devel/setup.bash # Allow installing dependencies (recognizing buzzmobile)
+  - rosdep install buzzmobile # Install dependencies for buzzmobile
```

Comment:
should we be installing dependencies before making? I'd assume we install deps in `before_script`, no?